src/router.js

```diff
@@ -15,20 +18,19 @@ export default function(app) {
     },
     events: {
       loaded: function(state, actions) {
-        addEventListener("popstate", function() {
+        match()
```

Comment:
Sorry to comment on such an old commit, but I'm curious why `match()` is being called here?  It's already called when the app starts up and the `state` object is instantiated.

Basically I was playing around with Hyperapp on Webpackbin.com, and logging various things.  Doing so, I noticed `match` gets called twice on app startup.  I commented out that line, and everything still seemed to work, but I'm sure the embedded browser window isn't perfectly representative of real use.